app/services/form_store.rb

```diff
@@ -28,6 +28,7 @@ class InvalidKeyError < StandardError; end
     degrees
     lead_school
     employing_school
+    lead_partner
```

Comment:
This will work for the transition from `lead_school` to `lead_partner`, however I suggest to leave a note to remove `lead_school` at [29](https://github.com/DFE-Digital/register-trainee-teachers/pull/4478/files#diff-27a93b900fa2c7b36720b11376f74028a5c998932b44e6c2b017a960b22ba5fbR29) after the migration to lead partners has been completed.